### PR TITLE
Feature/fbar 135 show hide filters button

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [FBAR-135]

This PR removes all mention of filters and tabs from TEC.

**Screencast:** Everything working as it did before

https://drive.google.com/file/d/1cAJZcEs7716jEHWiKSOblbDv647zgEUN/view?usp=sharing

[FBAR-135]: https://moderntribe.atlassian.net/browse/FBAR-135

### CODE REVIEW

File: src/views/v2/components/events-bar.php [hunk 1]
```diff
@@ -41,15 +41,11 @@
 		<?php $this->template( 'components/events-bar/search-button' ); ?>
 
 		<div
-			class="tribe-events-c-events-bar__search-filters-container"
-			id="tribe-events-search-filters-container"
-			data-js="tribe-events-search-filters-container"
+			class="tribe-events-c-events-bar__search-container"
```
Comment (lucatume) [inline src/views/v2/components/events-bar.php:44]: This change will break back-compat with existing CSS modifications, was this intentional?

File: src/views/v2/components/events-bar/filters.php [hunk 2]
```diff
@@ -1,15 +0,0 @@
-<?php
```
Comment (lucatume) [inline src/views/v2/components/events-bar/filters.php:1]: Same as above, removing this file will create issues with back-compatibility, intentional?

**Review by lucatume — COMMENTED** [review]
I have questions about back-compatibility and how updating or removing CSS classes or templates might affect existing overrides and customizations.

File: src/views/v2/components/events-bar.php (hunk 1 above)
Comment (bordoni) [reply src/views/v2/components/events-bar.php:44]: To answer your question I think B20.09 will be a feature release for both TEC and Filterbar

**Review by lucatume — APPROVED** [review]

**Pull request merged**